src/app/zap-templates/zcl/data-model/chip/color-control-cluster.xml

```diff
@@ -24,6 +24,20 @@ limitations under the License.
     <field name="ColorTemperature" mask="0x10" />
   </bitmap>
 
+  <bitmap name="ColorControlOptionsBitmap" type="BITMAP8">
```

Comment:
So this change is exposing the new bitmap and enum in client APIs where they did not use to be exposed before.

It's probably safer, from an API stability viewpoint, to just put these enum/bitmap declarations in the color control header for now instead of adding them to XML, especially since they are not actually being _used_ in the XML.

Then when the spec gets sorted out we can put them back in the XML, with whatever names (and value names; the value names here don't necessarily match what the spec will end up doing) the spec has.

Reply:
I was thinking that we do want names available to API clients - although this would indeed force the spec to pick this particular name if it was ever fixed. 

Do we have precedent of the "manually declare" approach? I am concerned that the manual approach will become stale compared to what code generator would generate.

Reply:
The spec update I'm working on have the following names for the mentioned data types:

- OptionsBitmap
- ColorModeEnum
- EnhancedColorModeEnum

As far as I understand EnhancedColorMode is an enum, but it also looks like this has been adjusted for this PR. Looks like it is only OptionsBitmap that would need to be aligned.

Reply:
> Do we have precedent of the "manually declare" approach?

Yes, for some of the "derive from a thing" clusters where ZAP support was lacking.

That said, we can obviously just expose stuff to API clients _if we are sure it has the names we will want_.

Note that this is not just about the names of the data types, but also about the names of the values in them.